src/contracts/interfaces/IRwaATokenManager.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IRwaATokenManager
+ * @author Aave
+ * @notice Defines the basic interface for the RWA aToken manager
+ */
+interface IRwaATokenManager {
+  /**
+   * @dev Grants authorized transfer role for an RWA aToken to an account
+   * @param aTokenAddress The address of the RWA aToken
+   * @param account The address of the new RWA aToken transfer role holder
+   */
+  function grantAuthorizedTransferRole(address aTokenAddress, address account) external;
+
+  /**
+   * @dev Revokes the authorized transfer role for an RWA aToken from an account
+   * @param aTokenAddress The address of the RWA aToken
+   * @param account The address of the RWA aToken transfer role holder to be revoked
+   */
+  function revokeAuthorizedTransferRole(address aTokenAddress, address account) external;
+
+  /**
+   * @dev Performs an authorized transfer of RWA aTokens from one account to another
+   * @param aTokenAddress The address of the RWA aToken
+   * @param from The address from which the RWA aTokens are transferred
+   * @param to The address that will receive the RWA aTokens
+   * @param amount The amount of RWA aTokens to transfer
+   * @return True if the transfer was successful, false otherwise
+   */
+  function transferRwaAToken(
+    address aTokenAddress,
+    address from,
+    address to,
+    uint256 amount
+  ) external returns (bool);
+
+  /**
+   * @dev Checks if an account has the authorized transfer role
+   * @param aTokenAddress The address of the RWA aToken
+   * @param account The address to check
+   * @return True if the given address has the authorized transfer role for the RWA aToken, false otherwise
+   */
```

Comment:
```suggestion
   * @notice Returns whether the account holds the authorized transfer role for a specific RWA aToken.
   * @param aTokenAddress The address of the RWA aToken
   * @param account The address of the account
   * @return True if the given address has the authorized transfer role for the RWA aToken, false otherwise
   */
```